slangpy/tests/slangpy_tests/test_load_image.py

```
@@ -0,0 +1,148 @@
+import tempfile
+import os
+
+import slangpy as spy
+from slangpy.testing import helpers
+
+
+@pytest.fixture(scope="module")
+def device() -> spy.Device:
+    try:
+        return helpers.get_device(spy.DeviceType.cuda)
+    except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)